radicle-httpd/src/api/v1/projects.rs

```diff
@@ -378,6 +386,98 @@ async fn issues_handler(
     Ok::<_, Error>(Json(issues))
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct IssueCreation {
+    pub title: String,
+    pub description: String,
+    pub tags: Vec<Tag>,
+}
+
+// // Create a new issue
+// // `POST /projects/:project/issues`
+async fn issue_creation_handler(
+    Extension(ctx): Extension<Context>,
+    Path(project): Path<Id>,
+    AuthBearer(token): AuthBearer,
+    Json(issue): Json<IssueCreation>,
+) -> impl IntoResponse {
+    let sessions = ctx.sessions.read().await;
+    sessions.get(&token).ok_or(Error::Auth("Unauthorized"))?;
+
+    let storage = &ctx.profile.storage;
+    let signer = ctx.profile.signer().map_err(Error::from)?;
+    let author = signer.public_key();
+    let repo = storage.repository(project)?;
+    let mut issues = Issues::open(*author, &repo)?;
+    issues
+        .create(issue.title, issue.description, &issue.tags, &signer)
+        .map_err(Error::from)?;
+
+    Ok::<_, Error>(())
+}
+
+// // Modifies an issue, defined on `cobs::issue::Action`
+// // `PATCH /projects/:project/issues/:id/:action`
+async fn issue_modify_handler(
```

Comment:
```suggestion
async fn issue_update_handler(
```

Reply:
This has been removed, since I want to focus in this PR on authorize a session.